### PR TITLE
Addition of x-word-story and QOTD custom commands.

### PR DESCRIPTION
**Pull request by SpecialEliteSNP**
I've added a x-word-story cc which is a custom command that manages a story-channel where a certain amount of words is required. I've also added a QOTD custom command which just makes sure that no one tries to answer the Question Of The Day twice and sends a report of the previous QOTD.
Both custom commands keep track of some data, I have another cc for that that accesses that data for members to see their answers or story lines (or counts).

Currently in use in my server with over 2k members. There's also a counting cc and "funinfo" cc I have but that might interfere with the already existing counting cc. I could do a PR for that as well. You can check it out on my [github](https://github.com/SpecialEliteSNP/yagpdb-cc/tree/main/fun)

- [X] Code changes have been tested on an instance of YAGPDB, or there are no code changes
- [X] I have read and followed the [contribution guide](../CONTRIBUTING.md)

### CODE REVIEW

File: fun/qotd.go.tmpl [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{{/* SETUP:
```
Comment (jo3-l) [inline fun/qotd.go.tmpl:1]: newline please

File: fun/x-word-story.go.tmpl [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{{/* SETUP:
```
Comment (jo3-l) [inline fun/x-word-story.go.tmpl:1]: same thing, newline please

**Review by jo3-l — CHANGES_REQUESTED** [review]
lgtm other than one tiny formatting issue.

About counting, feel free to PR a replacement.

File: fun/countingListener.go.tmpl [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {{/*
-	Extremely simple and consise counting system. 
+	Extremely simple and consise counting system.
```
Comment (SpecialEliteSNP) [inline fun/countingListener.go.tmpl:2]: Didn't mean to do this but whatever, doesn't hurt having one less space haha

**Review by SpecialEliteSNP — COMMENTED** [review]
Reverted all my messed up commits, so it's ready to be merged now 😬

**Review by jo3-l — APPROVED** [review]

**Comment by jo3-l** [comment]
Thanks!

**Pull request merged**